shell/platform/windows/angle_surface_manager.h

```diff
@@ -23,7 +23,7 @@ namespace flutter {
 // destroy surfaces
 class AngleSurfaceManager {
  public:
-  AngleSurfaceManager();
+  AngleSurfaceManager(WindowsRenderTarget*);
```

Comment:
Please name the argument, and add a comment here describing its role (and lifetime requirement, since it's a raw pointer).

Reply:
Fixed in next commit.  The target is retained for the life of the AngleSurfaceManager object, if I should be using a weak_ptr or shared_ptr, rather than raw ptr please re-open this.